packages/cozy-client/src/store/queries.js

```diff
@@ -165,7 +165,12 @@ const query = (state = queryInitialState, action, documents) => {
     case RECEIVE_QUERY_RESULT: {
       const response = action.response
       if (!response.data) {
-        return state
```

Comment:
nit: I think it would be worth adding a comment here saying that `data` might be null when querying a single doc that does not exist